### PR TITLE
Feat/create wallet doc update

### PR DESCRIPTION
**Pull request by dafunction**
i added a new block for the transaction signing page 'Advanced Settings' and i'll admit i'm a bit unsure if what i wrote is correct regarding the 'Import Keys to RPC' option.  also open to removing that 'Advanced Settings' block if folks feel it's unnecessary or out of place.

### CODE REVIEW

**Review by Bangalisch — CHANGES_REQUESTED** [review]
disregard this.. i messed up on my comment. I did run it trough Grammarly tho :)

File: docs/CreateWallet.md [hunk 1]
```diff
@@ -3,13 +3,46 @@
...
 
-### Hot Wallet
-![Create Wallet](./img/hotwallet/HotWallet.png)
+### 🔥 Hot Wallet
 
 If you do not have an existing wallet, you can generate a new one within your BTCPay Server. Whether you have an existing wallet or not, the quickest way to get a wallet connected to your store is to create a new wallet. You can always replace it with an alternative wallet after receiving a few small payments to your server, if you just want to get your store ready quickly.
 
 This type of wallet is also necessary to use features such as [Payjoin](./Payjoin.md) and [Liquid](https://github.com/btcpayserver/btcpayserver/issues/1282).
 
+After creating a store, you'll be able to attach a wallet by first navigating to the sidebar or slide out menu and clicking/tapping the **Bitcoin** button under the **Wallets** header. Alternatively, you can find the option to **Set up a wallet** on the Dashboard.
```
Comment (Bangalisch) [inline docs/CreateWallet.md:12]:
```suggestion
After creating a store, you'll be able to attach a wallet by first navigating to the sidebar or slide-out menu and clicking/tapping the **Bitcoin** button under the **Wallets** header. Alternatively, you can find the option to **Set up a wallet** on the Dashboard.
```

File: docs/CreateWallet.md [hunk 2]
```diff
@@ -3,13 +3,46 @@
...
+
+![Main Menu](./img/hotwallet/MainMenu.png)
+
+For a hot wallet you'll focus on the **I don't have a wallet** section and clicking/tapping the **Create a new wallet** button.
```
Comment (Bangalisch) [inline docs/CreateWallet.md:16]:
```suggestion
You'll focus on the **I don't have a wallet** section for a hot wallet and click the **Create a new wallet** button.
```

File: docs/CreateWallet.md [hunk 3]
```diff
@@ -3,13 +3,46 @@
...
+
+![New Wallet](./img/hotwallet/CreateNewWallet.png)
+
+On the proceeding page there will be two options, and in this case we'll select the **Hot wallet** button.
```
Comment (Bangalisch) [inline docs/CreateWallet.md:20]:
```suggestion
There will be two options on the proceeding page, and in this case, we'll select the **Hot wallet** button.
```

File: docs/CreateWallet.md [hunk 4]
```diff
@@ -3,13 +3,46 @@
...
+
+![Create Wallet](./img/hotwallet/HotWallet.png)
+
+For most folks, the default options including **Address Type** (Segwit), should work well for most applications and it's recommended to not change it unless you're sure of what you're doing. The **Payjoin** feature is optional, and you can learn more about it at the link [above](#hot-wallet).
```
Comment (Bangalisch) [inline docs/CreateWallet.md:24]:
```suggestion
For most people, the default options, including **Address Type** (Segwit), should work well for most applications, and it's recommended not to change it unless you're sure of what you're doing. The **Payjoin** feature is optional, and you can learn more about it at the link [above](#hot-wallet).
```

File: docs/CreateWallet.md [hunk 5]
```diff
@@ -3,13 +3,46 @@
...
+
+#### ⚙️ Advanced Settings
+
+- Optional [BIP39](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki#from-mnemonic-to-seed) passphrase
+    
+    - You can add a passphrase for your hot wallet's mnemonic for an additional layer of security.
+
+- Import keys to RPC
+
+    - This is for more advanced applicaitons of BTCPay Server.  Importing your keys to RPC will allow for users to leverage [bitcoind Wallet RPCs](https://developer.bitcoin.org/reference/rpc/index.html#wallet-rpcs) on the imported wallet.
```
Comment (Bangalisch) [inline docs/CreateWallet.md:42]:
```suggestion
    - This is for more advanced applications of BTCPay Server.  Importing your keys to RPC will allow users to leverage [bitcoind Wallet RPCs](https://developer.bitcoin.org/reference/rpc/index.html#wallet-rpcs) on the imported wallet.
```

File: docs/CreateWallet.md [hunk 6]
```diff
@@ -28,13 +61,16 @@ When a new wallet is generated, BTCPay Server will show you a twelve word recove
 
 Once you’ve received funds to your wallet and you decide to spend them, you can sign the transaction automatically, all inside BTCPay Server.
```
Comment (Bangalisch) [inline docs/CreateWallet.md:62]:
```suggestion
Once you've received funds to your wallet and decide to spend them, you can automatically sign the transaction inside BTCPay Server.
```

File: docs/CreateWallet.md [hunk 7]
```diff
@@ -28,13 +61,16 @@ When a new wallet is generated, BTCPay Server will show you a twelve word recove
...
 
-1. In BTCPay Server, go to > Wallets > Manage > Send
+1. In BTCPay Server, go to > Wallets > Bitcoin > Send
 2. Fill in the Destination address and the Amount
-3. Select Sign the hot wallet
-4. Review the transaction
-5. Broadcast the transaction
+3. Adjust the transaction settings including fee rate, confirmation time preference, and if you'd like trasaction fees to be subtracted from the amount you're sending
```
Comment (Bangalisch) [inline docs/CreateWallet.md:66]:
```suggestion
3. Adjust the transaction settings, including fee rate, confirmation time preference, and if you'd like transaction fees to be subtracted from the amount you're sending
```

File: docs/CreateWallet.md [hunk 8]
```diff
@@ -45,16 +81,19 @@ Storing private keys on a public server comes with risks. This is similar to the
 
 As mentioned above, the create wallet functionality includes risk of funds being stolen in the case of the server or account being compromised. To mitigate this risk, we advise you to:
```
Comment (Bangalisch) [inline docs/CreateWallet.md:82]:
```suggestion
As mentioned above, the create wallet functionality includes the risk of funds being stolen if the server or account is compromised. To mitigate this risk, we advise you to:
```

File: docs/CreateWallet.md [hunk 9]
```diff
@@ -45,16 +81,19 @@ Storing private keys on a public server comes with risks. This is similar to the
...
 
 :::danger
 Do not give anyone else access to your server's SSH keys or server account credentials when using a hot wallet. Anyone with access to your account can spend the funds from your hot wallet. If you need to allow account access to employees, developers, etc. use an [existing wallet](ConnectWallet.md#connect-an-existing-wallet) instead.
 :::
 
-
-### Watch-only wallet
+### 👀 Watch-only wallet
 
 Similar to the hot wallet, the watch-only wallet can get your store connected to a wallet instantly. In contrast, this option does not store the private keys on the server. As a result the wallet becomes "watch-only" for any received funds.
```
Comment (Bangalisch) [inline docs/CreateWallet.md:93]:
```suggestion
Like the hot wallet, the watch-only wallet can instantly get your store connected to a wallet. In contrast, this option does not store the private keys on the server. As a result, the wallet becomes "watch-only" for any received funds.
```

File: docs/CreateWallet.md [hunk 10]
```diff
@@ -45,16 +81,19 @@ Storing private keys on a public server comes with risks. This is similar to the
...
+There are several routes you can take to spend funds with this type of wallet including importing the seed words into a hardware wallet to sign your transactions using the [BTCPay Server Vault application](https://docs.btcpayserver.org/Vault/), [PSBT](https://docs.btcpayserver.org/Wallet/#psbt), or the least recommended manually providing your seed words every time.
+
+![BTCPay Server Transaction Signing Options](./img/hotwallet/SignTransaction.png "BTCPay Server Transaction Signing Options")
+
+Alternatively you can spend funds in another external wallet where you have imported your BTCPay Server produced seed words.  If you do import your seed words into an external wallet, you can also use a PSBT to spend the funds, assuming the wallet supports it.  This option will be available on the wallet's send page.  Be sure to consider the [gap limit issue](./FAQ/Wallet.md#missing-payments-in-my-software-or-hardware-wallet) if you're using an external wallet with your watch-only wallet.
```
Comment (Bangalisch) [inline docs/CreateWallet.md:99]:
```suggestion
Alternatively, you can spend funds in another external wallet where you have imported your BTCPay Server-produced seed words.  If you import your seed words into an external wallet, you can also use a PSBT to spend the funds, assuming the wallet supports it.  This option will be available on the wallet's send page.  Be sure to consider the [gap limit issue](./FAQ/Wallet.md#missing-payments-in-my-software-or-hardware-wallet) if you're using an external wallet with your watch-only wallet.
```

**Review by Bangalisch — COMMENTED** [review]
Grammarly polishing :) 
Some typos and some rephrasing. 
Good job !! :D

File: docs/CreateWallet.md [hunk 11]
```diff
@@ -3,13 +3,46 @@
 - [Hot Wallet](#hot-wallet)
 - [Watch-only wallet](#watch-only-wallet)
 
-### Hot Wallet
-![Create Wallet](./img/hotwallet/HotWallet.png)
+### 🔥 Hot Wallet
```
Comment (Bangalisch) [inline docs/CreateWallet.md:6]:
```suggestion
###  Hot Wallet
```

File: docs/CreateWallet.md [hunk 12]
```diff
@@ -3,13 +3,46 @@
...
+
+![Create Wallet](./img/hotwallet/HotWallet.png)
+
+For most folks, the default options including **Address Type** (Segwit), should work well for most applications and it's recommended to not change it unless you're sure of what you're doing. The **Payjoin** feature is optional, and you can learn more about it at the link [above](#hot-wallet).
+
+Find out more about advanced settings [here](#⚙️-advanced-settings).
+
+![Wallet Settings](./img/hotwallet/WalletSettings.png)
+
+#### 🌱 Recovery Seed
```
Comment (Bangalisch) [inline docs/CreateWallet.md:30]:
```suggestion
####  Recovery Seed
```

File: docs/CreateWallet.md [hunk 13]
```diff
@@ -3,13 +3,46 @@
...
+
+The last step to creating a hot wallet is making sure you document your recovery seed.  After you've done so, tick the checkbox that says *I have written down my recovery phrase and stored it in a secure location* and click the **Done** button.
+
+#### ⚙️ Advanced Settings
```
Comment (Bangalisch) [inline docs/CreateWallet.md:34]:
```suggestion
####  Advanced Settings
```

File: docs/CreateWallet.md [hunk 14]
```diff
@@ -45,16 +81,19 @@ Storing private keys on a public server comes with risks. This is similar to the
...
+- Enable two factor or U2F authentication
+- Occasionally move funds to your cold storage either manually or by configuring [BTC Transmuter](https://github.com/btcpayserver/btcTransmuter/blob/master/README.md) with automatic payment forwarding.
 
 :::danger
 Do not give anyone else access to your server's SSH keys or server account credentials when using a hot wallet. Anyone with access to your account can spend the funds from your hot wallet. If you need to allow account access to employees, developers, etc. use an [existing wallet](ConnectWallet.md#connect-an-existing-wallet) instead.
 :::
 
-
-### Watch-only wallet
+### 👀 Watch-only wallet
```
Comment (Bangalisch) [inline docs/CreateWallet.md:91]:
```suggestion
###  Watch-only wallet
```

File: docs/CreateWallet.md [hunk 15]
```diff
@@ -3,13 +3,46 @@
...
+
+![New Wallet](./img/hotwallet/CreateNewWallet.png)
+
+On the proceeding page there will be two options, and in this case we'll select the **Hot wallet** button.
+
+![Create Wallet](./img/hotwallet/HotWallet.png)
+
+For most folks, the default options including **Address Type** (Segwit), should work well for most applications and it's recommended to not change it unless you're sure of what you're doing. The **Payjoin** feature is optional, and you can learn more about it at the link [above](#hot-wallet).
+
+Find out more about advanced settings [here](#⚙️-advanced-settings).
```
Comment (Bangalisch) [inline docs/CreateWallet.md:26]:
```suggestion
Find out more about advanced settings [here](#advanced-settings).
```

**Comment by dafunction** [comment]
Thanks for that @Bangalisch!
I've gone ahead and committed your changes to this branch.

**Comment by Bangalisch** [comment]
Awesome! 

@pavlenex might wanna do one more run over it ^^

File: docs/CreateWallet.md [hunk 16]
```diff
@@ -3,13 +3,46 @@
...
+
+For most people, the default options, including **Address Type** (Segwit), should work well for most applications, and it's recommended not to change it unless you're sure of what you're doing. The **Payjoin** feature is optional, and you can learn more about it at the link [above](#hot-wallet).
+
+Find out more about advanced settings [here](#advanced-settings).
+
+![Wallet Settings](./img/hotwallet/WalletSettings.png)
+
+####  Recovery Seed
+
+The last step to creating a hot wallet is making sure you document your recovery seed.  After you've done so, tick the checkbox that says *I have written down my recovery phrase and stored it in a secure location* and click the **Done** button.
```
Comment (pavlenex) [inline docs/CreateWallet.md:32]: Double space
```suggestion
The last step to creating a hot wallet is making sure you document your recovery seed. After you've done so, tick the checkbox that says *I have written down my recovery phrase and stored it in a secure location* and click the **Done** button.
```

**Review by pavlenex — APPROVED** [review]

**Review by pavlenex — APPROVED** [review]
Good additions thanks to both of you for going through it and improving. I'll merge this now, I feel like `#### Spending funds with BTCPay Hot Wallet` and watch-only wallet should be in the `wallets` page so perhaps adding those there would make more sense, since I think we already talk about spending steps on that page.

**Pull request merged**